src/components/variant/ui/header.tsx

```diff
@@ -44,23 +44,23 @@ export const VariantHeader = observer(
 
     const filteredNo = toJS(mainTableStore.filteredNo)
 
-    const canGetPrevVariant = (): boolean => {
-      return !(filteredNo[filteredNo.indexOf(variantStore.index) - 1] >= 0)
-    }
+    const canGetPrevVariant = !(
+      filteredNo[filteredNo.indexOf(variantStore.index) - 1] >= 0
+    )
 
-    const canGetNextVariant = (): boolean => {
-      return !(filteredNo[filteredNo.indexOf(variantStore.index) + 1] >= 0)
-    }
+    const canGetNextVariant = !(
```

Comment:
seems that `filteredNo[filteredNo.indexOf(variantStore.index) + 1] < 0` is simpler

Reply:
Yep, you are right